pipelines/provision/postgres-db-update/Jenkinsfile

```diff
@@ -20,7 +20,7 @@ node() {
                 module = sh(returnStdout: true, script: "echo $JOB_NAME").split('/')[-2].trim()
                 jobName = sh(returnStdout: true, script: "echo $JOB_NAME").split('/')[-1].trim()
                 ansiblePlaybook = "${currentWs}/ansible/postgresql-data-update.yml"
-                ansibleExtraArgs = "--vault-password-file /var/lib/jenkins/secrets/vault-pass -v"
+                ansibleExtraArgs = "--vault-password-file /var/lib/jenkins/secrets/vault-pass -vvvv"
```

Comment:
@SMYALTAMASH Please change verbosity to 2

Reply:
removed verbose as not required